source/_components/sensor.awair.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: page
+title: "Awair Air Quality Monitor"
+description: "Instructions on how to setup Awair devices in Home Assistant."
+date: 2018-11-18 20:41
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: awair.jpg
+ha_category: Health
+ha_release: 0.83
+ha_iot_class: "Cloud Polling"
+---
+
+The `awair` sensor platform will fetch data from your [Awair device(s)](https://getawair.com).
+
+You will need to request access to the Awair API and obtain an access token from the Awair [Developer Console](https://developer.getawair.com/).
+
+## {% linkable_title Configuring the Platform %}
+
+To enable these sensors, add the following lines to your `configuration.yaml` file:
+
+```yaml
+sensor:
+  - platform: awair
+    access_token: ACCESS_TOKEN
+```
+
+The Awair API has stringent usage quotas. The API method to discover devices in your account is
+limited to only 6 calls per 24 hours. If you find that you've exceeded this quota, you may optionally
+append device information to your configuration to bypass this call:
+
+```yaml
+sensor:
+  - platform: awair
+    access_token: ACCESS_TOKEN
+    devices:
+      - uuid: UUID
+```
+
+{% configuration %}
+access_token:
+  description: The access token for the Awair API.
+  required: true
+  type: string
+devices:
+  description: An optional list of devices to manually configure rather than relying upon API discovery.
```

Comment:
... to manually configure devices rather than...

Reply:
I changed it a bit - I wasn't exactly sure what you were trying to express, though.